test/helpers/signers.js

```diff
@@ -150,10 +150,6 @@ class RSASigningKey {
 }
 
 class RSASHA256SigningKey extends RSASigningKey {
-  static random(modulusLength = 2048) {
-    return new RSASHA256SigningKey(generateKeyPairSync('rsa', { modulusLength }));
-  }
-
```

Comment:
why remove that part ?

Reply:
Because it's repeated code from the `RSASHA256SigningKey`, it's still available now that we're instantiating the signers with `this`